backend/app/schemas.py

```
@@ -106,53 +122,51 @@
-    order_items: List[OrderItemCreate]
+class OrderOut(BaseModel):  
+    order_id: int  
+    user_id: int  
+    status: str  
+   
+  
+    class Config:  
+        from_attributes = True
 
+class OrderUpdate(BaseModel):
+    status: st
```

Ruff (0.8.2)

Undefined name `st`

(F821)